apps/teams-test-app/e2e-test-data/profile.json

```diff
@@ -23,9 +23,9 @@
           "width": 0,
           "height": 0
         },
-        "triggerType": "MouseClick"
+        "triggerType": "MouseHover"
```

Comment:
This reminds me to check on web hub sdk, where profile is being supported.
 
It also needs a fix there?
https://office.visualstudio.com/ISS/_git/metaos-hub-sdk?path=%2Fpackages%2Fmetaos-hub-sdk%2Fsrc%2FServices%2FMessageHandling%2Fprofile-message-handler.test.ts&_a=contents&version=GBmain

@Kangxuan-Ye  @TrevorJoelHarris could you take a look and comment? thanks